coq-builtin.elpi

```diff
@@ -1797,14 +1797,29 @@ type include-module-type modtypath -> synterp-action.
 type import-module modpath -> synterp-action.
 type export-module modpath -> synterp-action.
 
+% Synterp action group
+kind group type.
+type grp int -> string -> group.
```

Comment:
Should this be a typeabbrev to some `ctype "group"` without any constructors?

Reply:
Absolutely,the type should be opaque. See the OpaqueData module in the API

Reply:
Done.